Source/WebCore/platform/graphics/gstreamer/mse/AppendPipeline.cpp

```diff
@@ -851,6 +852,35 @@ Ref<WebCore::TrackPrivateBase> AppendPipeline::makeWebKitTrack(int trackIndex)
     return track.releaseNonNull();
 }
 
+
+void AppendPipeline::Track::emplaceOptionalParserForFormat(AppendPipeline* appendPipeline, GstBin* bin, const GRefPtr<GstCaps>& newCaps)
+{
+    // Some audio files unhelpfully omit the duration of frames in the container. We need to parse
+    // the contained audio streams in order to know the duration of the frames.
+    // This is known to be an issue with YouTube WebM files containing Opus audio as of YTTV2018.
+    // If no parser is needed, a GstIdentity element will be created instead.
+
+    if (parser) {
```

Comment:
Unrelated to this PR, at some point member variables should be renamed with `m_` prefix. Current status is confusing.

Reply:
That would only apply to real classes, not structs, though I agree with you, see no real difference and should rename.